sources/packages/forms/src/form-definitions/confirmsstudentenrollment.json

```diff
@@ -1,4101 +1,3458 @@
+              "tableView": false,
+              "components": [
+                {
+                  "label": "Program Details Label",
+                  "attrs": [{ "attr": "", "value": "" }],
+                  "content": "Program Details",
+                  "refreshOnChange": false,
+                  "customClass": "font-weight-bold h4",
+                  "key": "programDetailsLabel",
+                  "type": "htmlelement",
+                  "input": false,
+                  "tableView": false,
+                  "placeholder": "",
+                  "prefix": "",
+                  "suffix": "",
+                  "multiple": false,
+                  "defaultValue": null,
+                  "protected": false,
+                  "unique": false,
+                  "persistent": false,
+                  "hidden": false,
+                  "clearOnHide": true,
+                  "refreshOn": "",
+                  "redrawOn": "",
+                  "modalEdit": false,
+                  "dataGridLabel": false,
+                  "labelPosition": "top",
+                  "description": "",
+                  "errorLabel": "",
+                  "tooltip": "",
+                  "hideLabel": false,
+                  "tabindex": "",
+                  "disabled": false,
+                  "autofocus": false,
+                  "dbIndex": false,
+                  "customDefaultValue": "",
+                  "calculateValue": "",
+                  "calculateServer": false,
+                  "widget": null,
+                  "attributes": {},
+                  "validateOn": "change",
+                  "validate": {
+                    "required": false,
+                    "custom": "",
+                    "customPrivate": false,
+                    "strictDateValidation": false,
+                    "multiple": false,
+                    "unique": false
+                  },
+                  "conditional": { "show": null, "when": null, "eq": "" },
+                  "overlay": {
+                    "style": "",
+                    "left": "",
+                    "top": "",
+                    "width": "",
+                    "height": ""
+                  },
+                  "allowCalculateOverride": false,
+                  "encrypted": false,
+                  "showCharCount": false,
+                  "showWordCount": false,
+                  "properties": {},
+                  "allowMultipleMasks": false,
+                  "tag": "p",
+                  "id": "ezkcpn",
+                  "addons": []
+                },
+                {
+                  "label": "programName",
+                  "tag": "p",
+                  "className": "",
+                  "attrs": [{ "attr": "", "value": "" }],
+                  "content": "{{ data.applicationProgramName }}",
+                  "refreshOnChange": true,
+                  "customClass": "font-weight-bold color-blue h4",
```

Comment:
is h4? then why `font-weight-bold`? is this style correct?

Reply:
Not part of my Changes.